lib/app/presenter/ui/conversion/widgets/amount_form_field.dart

```diff
@@ -18,7 +18,7 @@ class CustomFormField extends HookConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
-    bool valid = ref.watch(isValidProvider);
+    String initialValue = '';
```

Comment:
n precisa desse initialValue porque padrão é vazio

Reply:
Estou com um problema no formField, quando o usuario inicia digitando "." (ponto), estoura um erro. 
Provisoriamente criei uma logica para se for iniciado a digitação com ponto, ele seta o inicialValue para 0, mas já
estou estudando uma forma melhor de resolver isso.